src/common/constants/index.ts

```diff
@@ -1287,21 +1287,67 @@ export function getBloodTestAnalysisPrompt(data: CreateReviewDataDto): string {
     )
     .join('\n');
 
+  const calculateStatus = (value: number, min: number, max: number): string => {
```

Comment:
1) file for 1289 strings not ok, please check how logically they can be separated to multiple files. 
2) add comments to each method on how data looked before and after the function. and add test
3)  calculateStatus is not a constant, its util function , move it to utils (if such folder dont exist create it in the root)